tests/search_test.go

```diff
@@ -311,6 +332,23 @@ func TestSearchSentBefore(t *testing.T) {
 	})
 }
 
+func TestSearchSentSinceAndSentBefore(t *testing.T) {
+	// The result of this test should be no messages in the search result. Due to timezone adjustments, by
+	// mail.ParseDate the date was being converted to 17 Feb 2003 22:29:37 +000, causing the search to pass
+	// rather than fail.
+	runOneToOneTestWithAuth(t, defaultServerOptions(t), func(c *testConnection, _ *testSession) {
+		c.doAppend("INBOX", "Date: 18 Feb 2003 00:29:37 +0200\n\nTo: foo@foo.com\r\n")
+		c.C(`A002 SELECT INBOX`)
+		c.Se(`A002 OK [READ-WRITE] SELECT`)
+
+		t := time.Now()
+
+		c.C(fmt.Sprintf(`A001 search since %s sentbefore 18-Feb-2003`, t.Format("_2-Jan-2006")))
```

Comment:
so what happens if message has 

`Date: 18 Feb 2003 22:29:37 -0200`

and I ask

`search sentbefore 18-Feb-2003`

it should return empty search again, right? 

But for both dates `18 Feb 2003 00:29:37 +0200` and `Date: 18 Feb 2003 22:29:37 -0200` 
the search

 `search sentbefore 19-Feb-2003`
 
 is non empty, right? 
 
 Is it worth to add those cases?